build-system/eslint-rules/no-import.js

```diff
@@ -14,13 +14,13 @@
  * limitations under the License.
  */
 'use strict';
- const imports = ['sinon'];
- module.exports = function(context) {
+const imports = ['sinon'];
+module.exports = function(context) {
   return {
     ImportDeclaration(node) {
       const name = node.source.value;
       if (imports.includes(name)) {
-        context.report(node, `Importing ${name} is forbidden.`);
+        context.report({node, message: `Importing ${name} is forbidden.`});
```

Comment:
Is this a correct change? from two arguments to one?

Reply:
It is. `context.report` takes an object.

See https://eslint.org/docs/developer-guide/working-with-rules#contextreport
Also see https://github.com/ampproject/amphtml/search?utf8=%E2%9C%93&q=context.report